### DISCUSSION TITLE
Simple RSS Mastodon Bot - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#17] [General]
# Simple RSS Mastodon Bot - Alan Pope's blog

Linux Matters I recently started presenting Linux Matters podcast with my friends Martin Wimpress and Mark Johnson.
In episode 4 (that link will only work once the episode is released) I briefly talked about some simple bots I setup on the Ubuntu Social Mastodon instance (which, incidentally I talked about in episode 1).
This blog post accompanies episode 4. Linux Matters is part of the Late Night Linux (LNL) family. If you support us on the LNL Patreon, you’ll often get the episode delivered early, as well as advert free.

https://popey.com/blog/2023/05/simple-rss-mastodon-bot/

**Comment by sunjam** [comment]
**[feed2toot](https://feed2toot.readthedocs.io/en/latest/)**.  Been around forever and coded in Python.  I just want to give them the attention they deserve since their project has offered this in Python for a very long time, so let's give them a nod.  :)  https://feed2toot.readthedocs.io/en/latest/
